domino_environments/_environments.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, host: str):
     def deployment_version(self):
         return self.host + "/version"
 
+    def get_environments_list(self):
```

Comment:
I would like to keep the method names consistent
```suggestion
    def environments_list(self):
```